templates/e2e-llm-workflows/README.ipynb

```diff
@@ -917,7 +941,8 @@
    ],
    "source": [
     "# Load test set for eval\n",
-    "ft_test_ds = ray.data.read_json(f\"{os.environ['ANYSCALE_ARTIFACT_STORAGE']}/viggo/test.jsonl\")\n",
+    "test_path = config_fields[\"test_path\"]\n",
+    "ft_test_ds = ray.data.read_json(test_path)\n",
     "test_data = ft_test_ds.take_all()\n",
     "test_data[0]\n"
    ]
```

Comment:
another thing to demonstrate the use of datasets is to write eval results back to a dataset that can be used after to calculate metrics

Reply:
will address in separate PR, see [PR descr.](https://github.com/anyscale/templates/pull/313#issue-2503507658)

> - improve the template flow
>   - eg. removing the need to copy/paste values (eg. model_id) from logs